internal/constellation/helm/generateCertManager.sh

```diff
@@ -18,4 +18,20 @@ helm pull cert-manager \
   --untar \
   --untardir "charts"
 
+echo "Pinning cert-manager images..."
```

Comment:
I assume that yq already formats the helm charts. Since those values are exposed as helm values, I favor just setting them like all the other helm values.

Reply:
I don't like that approach because we then separate the helm version (set in the script) from the according hashes.

Reply:
Setting the hashes later (i.e. "like all the other helm values") would allow us do update the hashes using renovate.
We already do this for the cluster-autoscaler and cloud-controller-manager images, and some others.
See our [versions.go](https://github.com/edgelesssys/constellation/blob/a1dbd13f9577e23d99012416ab1b97c83ecfdedb/internal/versions/versions.go) file.
Not 100% how/if we want that for charts we pull in as third party dependencies since the versions should usually not be updated for one version of a chart, but it could make things easier since we wouldn't need to manually recalculate the image hashes every time we update the chart

Reply:
I assume @burgerdev motivation was to protect against a malicious version overwrite in the registry?
I agree that it would be good to use renovate.

Reply:
I'm facing more or less the same issue in my other hash pinning attempts, and this is the approach that I feel most comfortable with so far:

1. We want to specify a (chart|k8s|.*) version `$SOMEWHERE`, which ideally would be renovated.
2. As a consequence of that version choice, a list of required container images emerges.
3. We automatically derive the image hashes for those
4. and put them `$SOMEWHERE`.

The place for (1) is currently this script. I think that it would make sense to do (2-4) here as well, at least in the short term. If we can come up with 

* a place for storing (1) 
  - _globally_
* a generator with input (1) and output (2) 
  - _per use case_
* a generator with input (2) and output (3) 
  - _globally_
* a generator with input (3) 
  - _for a couple of different use cases_, e.g. `versions.go`, `values.go`

life would be a lot easier, but I think that's outside the scope of this change. However, I do think it's worthwile considering.

Long story short, +1 to keeping the chart version and the image hashes close together.